backend/fms_core/experiments.py

```diff
@@ -1,4 +1,4 @@
-PROTOCOLS_BY_EXPERIMENT_TYPE_NAME = {
+PROTOCOLS_BY_RUN_TYPE_NAME = {
```

Comment:
Do we still need this dictionary if it is to be stored in a migration?

Reply:
I decided not to do real changes just to convert the model... This will go away in another PR... but yeah this is part of the things that will go away...